src/v1/protocol/RootstockProtocol.ts

```diff
@@ -29,24 +35,61 @@ class RootstockProtocolImpl extends RootstockBaseProtocolImpl implements Rootsto
 
     const baseProtocolOptions: EthereumBaseProtocolOptions<RootstockUnits, RootstockProtocolNetwork> = {
       network: completeOptions.network,
-      identifier: MainProtocolSymbols.ROOTSTOCK,    
+      identifier: MainProtocolSymbols.ROOTSTOCK,
       name: 'Rootstock',
       standardDerivationPath: `m/44'/137'/0'`,
       units: DEFAULT_ROOTSTOCK_UNITS_METADATA,
       mainUnit: 'RBTC',
-      // https://stats.rsk.co/
-      feeDefaults: {
-        low: newAmount(1244.04 /* 21000 GAS * 0.05924 Gwei */, 'ETH').blockchain(DEFAULT_ROOTSTOCK_UNITS_METADATA),
-        medium: newAmount(1866.06 /* 21000 GAS * 0.05924 * 1.5 Gwei */, 'ETH').blockchain(DEFAULT_ROOTSTOCK_UNITS_METADATA),
-        high: newAmount(2488.08 /* 21000 GAS *  0.05924 * 2 Gwei */, 'ETH').blockchain(DEFAULT_ROOTSTOCK_UNITS_METADATA)
-      }
+      feeDefaults: RootstockProtocolImpl.getFeeUnits('RBTC')
     }
-    console.log('baseProtocolOptions', baseProtocolOptions.feeDefaults?.low.unit);
 
     const ethereumProtocol = new EthereumBaseProtocolImpl(nodeClient, infoClient, baseProtocolOptions)
 
     super(ethereumProtocol, nodeClient, infoClient, completeOptions)
   }
+
+  public static getFeeUnits(unit) {
```

Comment:
What type is `unit` of? The values are static and the comments suggest their unit is already defined (`GWEI`), so does it have to be parametrized?